jobs/keepalived/templates/bin/action.sh.erb

```diff
@@ -32,5 +37,23 @@ if [[ "$iaas_name" == "aws" ]]; then
     sudo ip addr del $virtual_ip/32 dev eth0
     set -e
   fi
+elif [[ "$iaas_name" == "openstack"  ]]; then
+  source /var/vcap/jobs/keepalived/bin/utils/openstack/generate_token
+  source /var/vcap/jobs/keepalived/bin/utils/openstack/port_operations
+  this_port_id=$(get_port_id ${SOURCE_IP})
+  if [[ "$STATE" == "MASTER" ]]; then
+    port_update_response=$(update_port ${this_port_id})
```

Comment:
Please run a shellcheck for the script. You might end up having violations and it's better to fix them here